test/parallel/test-inspector-has-idle.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+const { Session } = require('inspector');
+const { promisify } = require('util');
+
+const sleep = promisify(setTimeout);
+
+async function test() {
+  const inspector = new Session();
+  inspector.connect();
+
+  inspector.post('Profiler.enable');
+  inspector.post('Profiler.start');
+
+  await sleep(10);
```

Comment:
I recommend to use a significant longer wait time here or move this test into sequential. Some CI machines are that frequently to busy to schedule the sampling thread reliable in 10ms.

Reply:
Extended to 1000ms. Not sure if there's a more reasonable duration, or perhaps a better way to structure the tests to avoid the longer timeout. Hopefully that's good enough. 🤔

Reply:
That should be enough. If it turns out that test is still flaky we can move it to sequential later.